docs/source/conf.py

```diff
@@ -30,7 +31,7 @@
 # You can specify multiple suffix as a list of string:
 #
 # source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
+source_suffix = ['.rst', '.md']
```

Comment:
The whole comment above can be removed.

Reply:
It was there originally. Maybe let's leave it there for convenience in the future?..

Reply:
We can remove it :)

Reply:
woohooa)